Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj

```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with the MonoGame Framework Content Builder (MGCB) tool used for packaging content in to xnb files.</Description>
```

Comment:
```suggestion
    <Description>The MonoGame Content Builder (MGCB) tool used for packaging content into xnb files.</Description>
```

Reply:
I would suggest:

`The MonoGame Content Builder (MGCB) command line tool used for optimizing content for runtime use.`